frontend/src/components/homepage/Home.tsx

```diff
@@ -1,6 +1,13 @@
-export const Home: React.FunctionComponent = () => (
-  <>
-    <h1>Whale Spotting</h1>
-    <p>Spot whales!</p>
-  </>
-);
+import React, { useState } from "react";
+import internal from "stream";
+import "./Home.scss";
+
+import { NewSightings } from "../new-sightings-page/NewSightings";
+
+export const Home: React.FunctionComponent = () => {
+  return (
+    <div>
+      <NewSightings />
+    </div>
```

Comment:
Don't use a div here!
```suggestion
    <NewSightings />
```